blosc2/core.py

```diff
@@ -603,8 +603,9 @@ def os_release_pretty_name():
                     return value
         except IOError:
             pass
-    else:
-        return None
+        finally:
+			f.close()
```

Comment:
Too much indentation

Reply:
Rather tabs instead of spaces, sorry about that.